src/main/java/com/nvs/store/controllers/OrderController.java

```diff
@@ -0,0 +1,12 @@
+package com.nvs.store.controllers;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+@RequiredArgsConstructor
+public class OrderController {
```

Comment:
Please remove this controller from commit if it is not scope of this pull request